tests/scanner/audit/iam_rules_engine_test.py

```diff
@@ -57,6 +56,26 @@ def setUp(self):
             display_name='My project 3',
             parent=self.folder1)
 
+        self.mock_org_policy_resource = mock.MagicMock()
+        self.mock_org_policy_resource.full_name = (
+            'organization/778899/')
+
+        self.mock_folder1_policy_resource = mock.MagicMock()
+        self.mock_folder1_policy_resource.full_name = (
+            'organization/778899/folder/333/')
+
+        self.mock_project1_policy_resource = mock.MagicMock()
+        self.mock_project1_policy_resource.full_name = (
+            'organization/778899/project/project1/my-project-1')
+
+        self.mock_project2_policy_resource = mock.MagicMock()
+        self.mock_project2_policy_resource.full_name = (
+            'project2/my-project-2')
+
+        self.mock_project3_policy_resource = mock.MagicMock()
+        self.mock_project3_policy_resource.full_name = (
+            'organization/778899/folder/333/project/my-project-3')
+
         # patch the organization resource relation dao
         self.patcher = mock.patch('google.cloud.forseti.common.data_access.org_resource_rel_dao.OrgResourceRelDao')
```

Comment:
This dao should no longer be called, so it should no longer need to be mocked, right?